SignalMessaging/utils/OWSWindowManager.m

```diff
@@ -184,8 +184,9 @@ - (UIWindow *)createCallViewWindow:(UIWindow *)rootWindow
     UIViewController *viewController = [OWSWindowRootViewController new];
     viewController.view.backgroundColor = [UIColor ows_materialBlueColor];
 
-    OWSNavigationController *navigationController =
-        [[OWSNavigationController alloc] initWithRootViewController:viewController];
+    // NOTE: Do not use OWSNavigationController for call window.
```

Comment:
Why not? 

In general prefer comments that explain why vs. what

Reply:
Improved the comment here:

https://github.com/signalapp/Signal-iOS/commit/c71081c8758c94e51859d75e56649f42e2f62990